engine/grid.py

```diff
@@ -206,14 +206,14 @@ def get_neighbor_node(self, row, col, row_max, col_max):
            return neighbor_node
 
     ##Activates rectangle based on row start/end, col start/end
-    def activate_rectangle(nodes, row, col, row_limit, col_limit):
+    def activate_rectangle(self, row, col, row_limit, col_limit):
         """
         Activates all the nodes in a rectangle.
         """
         for x in range(row, row_limit):
             for y in range(col, col_limit):
-                nodes[x,y].is_active = True
-        return nodes
+                self.nodes[x,y].is_active = True
+        return self.nodes
 
```

Comment:
hmm can we take these changes out of this pr? i think its from the grid refactoring branch?